src/main/java/teammates/common/datatransfer/NotificationType.java

```diff
@@ -0,0 +1,77 @@
+package teammates.common.datatransfer;
+
+import teammates.common.util.Const;
+
+/**
+ * Represents the type of target user groups for notifications.
+ */
+public enum NotificationType {
+
+    /**
+     * Notes for maintenance.
+     */
+    MAINTENANCE(Const.NotificationType.MAINTENANCE),
+
+    /**
+     * Notes for version release.
+     */
+    VERSION_NOTE(Const.NotificationType.VERSION_NOTE),
+
+    /**
+     * Notes for deprecation.
+     */
+    DEPRECATION(Const.NotificationType.DEPRECATION),
+
+    /**
+     * Usage tips.
+     */
+    TIPS(Const.NotificationType.TIPS);
+
+    private final String notificationType;
+
+    NotificationType(String notificationType) {
+        this.notificationType = notificationType;
+    }
+
+    public String getNotificationType() {
+        return notificationType;
+    }
+
+    /**
+     * Formats the participant type as a singular noun.
+     *
+     * @return A user-friendly {@code String} representing this participant in singular form.
+     */
+    public String toSingularFormString() {
+        switch (this) {
+        case MAINTENANCE:
+            return "maintenance";
```

Comment:
Query on the hardcoding of `maintenance ` instead of using `Const.NotificationType.MAINTENANCE`